mix.exs

```diff
@@ -32,6 +38,7 @@ defmodule AWS.Mixfile do
       {:earmark, "~> 1.4", only: [:dev]},
       {:ex_doc, "~> 0.24", only: [:dev]},
       {:bypass, "~> 2.1", only: [:test]},
+      {:finch, "~> 0.14.0 or ~> 0.14.0", optional: true},
```

Comment:
Shouldn't this be `~> 0.13.0 or ~> 0.14.0`:

```suggestion
      {:finch, "~> 0.13.0 or ~> 0.14.0", optional: true},
```

Reply:
Yep, sorry 🤦

(Btw, the compatibility goes beyond that but I kept these just to keep it simple yet friendly to upgrades)